test/Dialect/FIRRTL/SFCTests/signal-mappings.fir

```diff
@@ -0,0 +1,74 @@
+; RUN: firtool %s --annotation-file %S/signal-mappings-subCircuit.json --firrtl-grand-central --verilog | FileCheck %s
```

Comment:
What is the utility in having the JSON file separate from the test? Is this to make it easier to test with SFC? If there's no major benefit, then it may be better to inline this into the FIRRTL file.

Reply:
The intention was to represent more or less closely what we'll have to do in our build flows, since this is an integration test. I tried to encode the fact that this comes from the `subCircuit.json` input file. But I totally agree, it's a bit far-fetched and this could just as well be an inline list of annotations. I'm happy to have it either way; let me know if you have any preferences!

Reply:
I've started playing with this for the GCT Views and I think it's better to keep it the way you have. Primarily because this allows us to play with different annotation files being applied to the same circuit in a more seamless way than sticking the annotations in the circuit. (E.g., you can easily test it with or without extraction by putting the extraction information into a separate file.)

Please leave it the way you have it! 😄

Reply:
Aye! :grin: